app/src/app/(auth)/sign-in.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useState } from "react";
 import Button from "../../components/Button";
 import Colors from "../../constants/Colors";
 import { Stack } from "expo-router";
-import { supabase } from "@/lib/supabase";
```

Comment:
dunno why these `@/` imports suddenly broke in my env but using lib/superbase directly still works